payloads/v10/channel.ts

```diff
@@ -1977,7 +2064,10 @@ export enum ChannelFlags {
 /**
  * https://discord.com/developers/docs/interactions/message-components#message-components
  */
-export type APIMessageComponent = APIActionRowComponent<APIMessageActionRowComponent> | APIMessageActionRowComponent;
+export type APIMessageComponent =
+	| APIMessageActionRowComponent
+	| APIMessageTopLevelComponent
+	| APISectionAccessoryComponent;
```

Comment:
This type is confusing. Why is there an actionRow, toplevel, and then section accessory pulled out separately?

From a typing perspective, this should just be a list of all components, no?